async-raft/tests/fixtures/mod.rs

```diff
@@ -115,6 +116,98 @@ impl RaftRouter {
         metrics
     }
 
+    /// Get a handle to the storage backend for the target node.
+    pub async fn get_storage_handle(&self, node_id: &NodeId) -> Result<Arc<MemStore>> {
+        let rt = self.routing_table.read().await;
+        let addr = rt
+            .get(node_id)
+            .ok_or_else(|| anyhow::anyhow!("could not find node {} in routing table", node_id))?;
```

Comment:
```suggestion
            .with_context(|| format!("could not find node {} in routing table", node_id))?;
```

Let's stick with using anyhow's `Context` system as much as possible. In many cases, it greatly simplifies error propagation with easy static str context values.